docs/book/introduction.md

```diff
@@ -4,27 +4,178 @@ description: Welcome to ZenML!
+into your organization, allowing you to easily reuse code developed by your 
+colleagues:
+
+```python
+from my_organization.steps import data_loader_step, model_validation_step
+
+@step
+def my_model_trainer_step(data):
+    ...
+
+@pipeline
+def my_pipeline():
+  data = data_loader_step()
+  model = my_model_trainer_step(data)
+  eval_metric = model_validation_step(model)
+```
+{% endtab %}
+
+{% tab title="ZenML for ML Engineers" %}
+
+As an ML Engineer, ZenML empowers you to take ownership of the entire ML 
+lifecycle end-to-end. Adopting ZenML means fewer handover points and more 
+visibility on what is happening in your organization.
+
+**ML Lifecycle Management:** ZenML's abstractions enable you to manage
+sophisticated ML setups with ease. After you define your ML workflows as 
+[Pipelines](./getting-started/core-concepts.md#1-development) and your 
+development, staging, and production infrastructures as 
+[Stacks](./getting-started/core-concepts.md#2-execution), you can move 
+entire ML workflows to different environments in seconds.
+
+```bash
+zenml stack set staging
+python run.py  # test your workflows on staging infrastructure
+zenml stack set prod
+python run.py  # run your workflows in production
+```
+
+**Reproducibility:** ZenML enables you to painlessly reproduce previous results
+by automatically tracking and versioning stacks, pipelines, and output 
+artifacts. In the ZenML dashboard, you can get an overview of everything that 
+has happened. Try it out at [https://demo.zenml.io/](https://demo.zenml.io/)!
+
+<figure>
+<img src=".gitbook/assets/intro_dashboard.png" alt="ZenML Dashboard Overview" width="50%">
+</figure>
+
+**Automated Deployments:** With ZenML, you no longer need to upload custom 
+Docker images to the cloud whenever you want to deploy a new model to 
+production. Simply define your ML workflow as a ZenML pipeline, let ZenML 
+handle all the containerization, and have your model automatically deployed to 
+a highly scalable Kubernetes deployment service like 
+[Seldon](./user-guide/component-galery/model-deployers/seldon.md).
+
+```python
+from zenml.integrations.seldon.steps import seldon_model_deployer_step
+
+from my_organization.steps import data_loader_step, model_trainer_step
+
+
+@pipeline
+def my_pipeline():
+  data = data_loader_step()
+  model = model_trainer_step(data)
+  seldon_model_deployer_step(model)
+```
+
+{% endtab %}
+
+{% tab title="ZenML for MLOps / Platform Engineers" %}
+
+As an MLOps infrastructure expert, ZenML enables you to define, deploy, and 
+manage sophisticated production environments that are easy to share with your 
+colleagues.
+
+**Standardization:** With ZenML, you can standardize MLOps infrastructure and 
+tooling across your organization. Simply set up a ZenML server, register your 
+production environment as a ZenML stack, and invite your colleagues to run 
+their ML workflows on it:
+
+```bash
+zenml deploy  # Deploy ZenML
```

Comment:
We can also add deploying mlflow here as an example. I would even suggest moving the deployment part into a seperate tab altogether so:

```
zenml deploy  # Deploy ZenML on any cloud
zenml experiment-tracker deploy my_tracker --flavor=mlflow --cloud=gcp  # Deploy any MLOps component in seconds
```

```
zenml secret create mlflow --password=my_password  # Create secrets on supported backends
zenml experiment-tracker register my_mlflow_tracker --flavor=mlflow --password {{ mlflow.my_password }}  # register stacks with ease
zenml stack register production...  # Register your production environment
zenml stack share production  # Make it available to your colleagues
```

Reply:
I added a `zenml ... deploy` example here

Reply:
Secrets syntax feels a bit too detailed for an intro page IMO

Reply:
I replaced the extensibility section with a separate deployment section now